src/main/java/thePackmaster/cards/siegepack/ThinkTwice.java

```diff
@@ -0,0 +1,62 @@
+package thePackmaster.cards.siegepack;
+
+import com.evacipated.cardcrawl.mod.stslib.patches.FlavorText;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.cards.siegepack.FlavorConstants.*;
+
+public class ThinkTwice extends AbstractSiegeCard {
+    public final static String ID = makeID("ThinkTwice");
+    private static final int COST = 2;
+    private static final int BLOCK = 6;
+    private static final int UPGRADE_BLOCK = 3;
+    private static final int BLOCK_PER_ATTACKER = 5;
+    private static final int UPGRADE_BLOCK_PER_ATTACKER = 2;
+
+    public ThinkTwice() {
+        super(ID, COST, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseBlock = BLOCK;
+        baseMagicNumber = magicNumber = BLOCK_PER_ATTACKER;
+
+        FlavorText.AbstractCardFlavorFields.flavorBoxType.set(this, FLAVOR_BOX_TYPE);
+        FlavorText.AbstractCardFlavorFields.boxColor.set(this, FLAVOR_BOX_COLOR);
+        FlavorText.AbstractCardFlavorFields.textColor.set(this, FLAVOR_TEXT_COLOR);
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        blck();
+        //Gain 5 (7) block per enemy that intends to attack.
+        for (AbstractMonster mo : AbstractDungeon.getMonsters().monsters) {
+            if (!mo.isDeadOrEscaped()) {
+                if (mo.intent == AbstractMonster.Intent.ATTACK
+                        || mo.intent == AbstractMonster.Intent.ATTACK_BUFF
+                        || mo.intent == AbstractMonster.Intent.ATTACK_DEBUFF
+                        || mo.intent == AbstractMonster.Intent.ATTACK_DEFEND) {
+                    //Wiz.atb(new GainBlockAction(AbstractDungeon.player, AbstractDungeon.player, this.block));
+                    Wiz.doBlk(magicNumber);
+                }
+            }
+        }
+    }
+
+    public void triggerOnGlowCheck() {
+        this.glowColor = AbstractCard.BLUE_BORDER_GLOW_COLOR.cpy();
+        for (AbstractMonster m : AbstractDungeon.getCurrRoom().monsters.monsters) {
+            if (!m.isDeadOrEscaped() && m.getIntentBaseDmg() >= 0) {
+                this.glowColor = AbstractCard.GOLD_BORDER_GLOW_COLOR.cpy();
+                break;
+            }
+        }
+    }
+
+    @Override
+    public void upp() {
+        upgradeBlock(UPGRADE_BLOCK);
+        upgradeMagicNumber(UPGRADE_BLOCK_PER_ATTACKER);
```

Comment:
too strong, keep only the base increase

Reply:
Sure thing. Can I make the base a little higher then? Looking at other PM 2-cost uncommon Block card, they tend to gain +4 block on upgrade, or give less than 4 while increasing their second effect. "Think Twice" would end up giving less block than those against a single target, while not having any second effect (orb/card creation, Strength gain etc).

Reply:
yeah, +4 is what I had in mind